quay-enterprise/custom-cert-kube-apps.md

```diff
@@ -0,0 +1,41 @@
+## Using Helm Registry Plug-in with a self-signed certificate
+
+This document assumes you have deployed [Quay Enterprise with a self-signed certificate.][self-signed]
+
+[Appr](https://github.com/app-registry/appr) uses a [Python Requests library][python-requests], which trusts only a standard set of certificates by default. This prevents Helm from interacting with a Quay Enterprise instance that is using a self-signed certificate.
```

Comment:
missed this one - sorry. change to [Appr][appr], and add link to the bottom of the page.